lib/features/family/features/reflect/presentation/widgets/rule_card.dart

```diff
@@ -38,6 +38,7 @@ class RuleCard extends StatelessWidget {
         ),
       ),
       Container(
+        height: 366,
```

Comment:
just to be sure. Should this have a fix height for all rule cards?

Reply:
Yes! In the design they are all the same height and by putting it this way the "read out loud" now shows correctly again.